### PR TITLE
Feature: Lights & DirectLight

### PR DESCRIPTION
**Pull request by Tucchhaa**

### CODE REVIEW

File: src/objects/camera.ts [hunk 1]
```diff
@@ -22,11 +22,10 @@ export class Camera extends SpaceEntity {
 	near: number;
 	far: number;
 
-	constructor(config: CameraConfig = {}, position: Vector3 = Vector3.zero, direction: Vector3 = Vector3.backward) {
+	constructor(config: CameraConfig = {}, position: Vector3 = Vector3.zero) {
 		super();
 
 		this.setPosition(position);
-		this.setDirection(direction);
```
Comment (jeffydc) [inline src/objects/camera.ts:29]: @Tucchhaa Why was this removed?

Comment (Tucchhaa) [reply src/objects/camera.ts:29]: Setting direction for the camera is unstable. Remember direction doesn't have Z rotation. It's better to use camera.rotate
